Hi Jing,

I'm seeing this error:

```
======================================================================
ERROR: tests.test_pubmedScore (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_pubmedScore
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/travis/build/pontikos/phenopolis/tests/test_pubmedScore.py", line 5, in <module>
    import pubmedScore
ImportError: No module named pubmedScore
----------------------------------------------------------------------
Ran 3 tests in 0.254s
FAILED (errors=1)
The command "python -m unittest discover" exited with 1.
```

I'll merge the pull request but we need to fix this.

Thanks,

Niko.

--- a/tests/test_pubmedScore.py
+++ b/tests/test_pubmedScore.py
@@ -1,7 +1,7 @@
 
 import unittest
 import sys
-sys.path.append('../pre_analysis/pubmedScore')
+sys.path.append('pre_analysis/pubmedScore')
 import pubmedScore
 import time
 

